src/ui/components/ImageUploader.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { FC, useRef } from 'react';
+
+type WhpptImageUploaderProps = {
+  uploadImage: (any) => void;
+};
+
+export const WhpptImageUploader: FC<WhpptImageUploaderProps> = ({ uploadImage }) => {
+  const imageInputRef: { current: HTMLInputElement } = useRef();
+
+  const openImageInput = () => {
+    imageInputRef && imageInputRef.current && imageInputRef.current.click();
+  };
+
+  const selectFile = event => {
```

Comment:
can be selectFile: React.ChangeEventHandler<HTMLInputElement> = event => ...
This will give you a typed imageUploaded var. The type should be File that can then also be used in the uploadImage callback.